src/darker/tests/test_files.py

```diff
@@ -1,22 +1,33 @@
 """Test for the `darker.files` module."""
 
-import io
-from contextlib import redirect_stderr
 from pathlib import Path
-from unittest.mock import MagicMock, patch
+
+import pytest
 
 from darker import files
 
 
-@patch("darker.files.find_user_pyproject_toml")
-def test_find_pyproject_toml(find_user_pyproject_toml: MagicMock) -> None:
+@pytest.mark.kwparametrize(
+    dict(start="only_pyproject/subdir", expect="only_pyproject/pyproject.toml"),
```

Comment:
```suggestion
    {"start":"only_pyproject/subdir", "expect":"only_pyproject/pyproject.toml"},
```

Reply:
This is a style decision I've made when using `pytest-kwparametrize`: to avoid a jungle of double quotes and make test cases more readable, I use the `dict()` constructor, and silence linter warnings in each module.